docs/pages/internationalization-in-kaizen.mdx

```diff
@@ -0,0 +1,62 @@
+import { Meta } from '@storybook/blocks'
+import { LinkTo } from "../../storybook/components/LinkTo"
+
+<Meta title="Guides/Internationalization in Kaizen" />
```

Comment:
Should site under the `KaizenProvider` docs, cause its technically describing the API of `KaizenProvider`